src/utils/redirectless.js

```diff
@@ -0,0 +1,80 @@
+
+const allowedAuthRequestParameters = ['client_id', 'response_type', 'code_challenge', 'code_challenge_method',
+  'redirect_uri', 'scope', 'state', 'idp', 'pfidpadapterid', 'access_token_manager_id', 'aud', 'nonce', 'prompt',
+  'acr_values', 'max_age', 'login_hint', 'ui_locales', 'id_token_hint', 'claims_locales'];
+
+/**
+ * Call onAuthorizationSuccess when 'COMPLETED' state is reached.
+ *
+ * @param {JSON} state          The current state
+ * @param {JSON} configuration  The redirectless configuration object
+ */
+export function completeStateCallback(state, configuration) {
+  if (state.status === 'COMPLETED' && state.authorizeResponse) {
+    configuration.onAuthorizationSuccess(state.authorizeResponse);
```

Comment:
Do you think its worth logging or throwing an error if there is no authorizeResponse for some reason?

Reply:
We might cause issues with password change complete state if we do that. Console log is worthless too since we remove all those statements from the production build